request.go

```diff
@@ -49,6 +50,7 @@ type Request struct {
 	RawCallMsg     ethereum.CallMsg
 	BlockNumber    *big.Int
 	BlockHash      common.Hash
+	Overrides      *map[common.Address]gethclient.OverrideAccount
```

Comment:
um why do we need this to be a pointer to map? I think a map alone is enough

Reply:
`func (ec *Client) CallContract(ctx context.Context, msg ethereum.CallMsg, blockNumber *big.Int, overrides *map[common.Address]OverrideAccount) `

The function signature `CallContract` of `gethclient` used a pointer for this map, so I left it for compatible.

Fixed, removed this pointer, and the callContract should be

`resp, err = c.gethClient.CallContract(req.Context(), req.RawCallMsg, req.BlockNumber, &req.Overrides)
`